makeservices/makemysql

```diff
@@ -4,20 +4,54 @@ if [[ "$(hostname)" != "tsunami" && "$(hostname)" != "dev-tsunami" ]]; then
     exit 1
 fi
 
-PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real | tee /dev/tty | tail -n 1 | grep -Po '(?<=: )([0-9a-zA-Z]){24,}$')
+IGNORE_WP=false
+UNREADABLE_OUTPUT=false
+SILENT=""
+
+# Parse arguments down below
+while (( "$#" )); do
+    case "$1" in
+        -i|--ignore-wp)
+            IGNORE_WP=true
+            shift
+            ;;
+        -u|--unreadable-output)
+            UNREADABLE_OUTPUT=true
+            shift
+            ;;
+        -s|--silent)
+            YES="--silent"
+            shift
+            ;;
+        # I've seen -*|--*=), but idk what does the = serve
+        -*|--*)
+            # Output to stderr
+            echo "Error: Unsupported flag $1" >&2
+            exit 1
+            ;;
+    esac
+done
+
+PASS=$(sudo -u mysql /opt/share/utils/makeservices/makemysql-real $SILENT | tail -n 1)
```

Comment:
I'm not sure what's the point of --silent, since we're always only going to see the last line (because of tail). Maybe we could make it so that makemysql-real prints its messages to stderr so they aren't affected but the subshell and tail?

Reply:
That is what I was expecting (only printing the password), but I do not know if other calls in `makemysql-real` print anything to `stdout`, so `tail` was added to make sure.
That being said, I was considering about what if `makemysql-real` failed, without `grep` `PASS` would be some weird stuffs

Reply:
I've actually done that, but I didn't test `makemysql-real` and the things it call might output some other mysterious stuff ig